simplecpp.cpp

```diff
@@ -2832,9 +2832,20 @@ static void simplifyName(simplecpp::TokenList &expr)
                 if (alt)
                     continue;
             }
+            if (tok->next && tok->next->str() == "(") {
+                if (outputList) {
```

Comment:
to me it looks like you can more or less use `throw std::runtime_error("Undefined function-like macro in directive: " + tok->str() + "( ... )")` here and the proper syntax error will be written in the output list. but the location will not be correct then will it? would it be better to throw an other error?